### ISSUE TITLE
Re-Open: New-UDStepper icons not respecting theme colors in PSU 4.x #3143

### ISSUE THREAD

**Issue by donlent** [ironmansoftware/powershell-universal#3225]
### Version

4.2.14

### Severity

Visual

### Steps to Reproduce

Sorry, I have to open this issue again since it's a regress:
https://github.com/ironmansoftware/issues/issues/3143

--> The Icon Shape of the Stepper is completely gone now.

### Expected behavior

```console
The Icon shape should be put back like it was on pre-4.2.14, even if not stylable. I don't care if that's a problem.
```

### Actual behavior

```console
The Icon Shape is completely gone. This is not good from a visual standpoint.
```

**Comment by adamdriscoll** [comment]
This issue has been mentioned on **Ironman Software Forums**. There might be relevant details there:

https://forums.ironmansoftware.com/t/udstepper-icon-gradients-not-following-theme-colors-in-4-2-12-unable-to-override/10361/4

**Comment by adamdriscoll** [comment]
We have implemented #3143 properly. It will now follow the theme palette. We have also added -IconStyle and -DarkIconStyle to allow for easy customization of the stepper icon's styles directly on the step.
